src/rtree_intf.ml

```diff
@@ -43,6 +43,9 @@ module type Value = sig
 
   val envelope : t -> envelope
   (** Given a value, calculates the envelope for it. *)
+
+  val mindist: t -> envelope -> float
+  val minmaxdist: t -> envelope -> float
```

Comment:
Do these need to be exposed in the interface?

Reply:
I wanted to test these functions as well. Is that not necessary?